tox.ini

```diff
@@ -17,6 +17,7 @@ deps=
     pytest-cov
     hypothesis[numpy] !=3.5.1, !=3.5.0
     numpy
+    pandas
```

Comment:
Right, I wanted to do that but put pandas in the wrong section, thanks for the catch!